Source/WebCore/Modules/webcodecs/WebCodecsAudioDataAlgorithms.cpp

```diff
@@ -75,7 +75,7 @@ bool isAudioSampleFormatInterleaved(const AudioSampleFormat& format)
     return false;
 }
 
-size_t computeBytesPerSample(const AudioSampleFormat& format)
+size_t computeBytesPerSample(AudioSampleFormat format)
```

Comment:
```suggestion
// https://w3c.github.io/webcodecs/#compute-copy-element-count
size_t computeBytesPerSample(AudioSampleFormat format)
```
Maybe we should rename the function to computeCopyElementCountInternal?

Reply:
I'm not sure. This comment doesn't apply to that method.
bytePerSample has little to do with computeCopyElementCount

And computeCopyElementCount is the spec name, and it does exactly the same computation as per the spec